packages/react/src/hooks/useManageSubscription.ts

```diff
@@ -37,35 +37,37 @@ export const useManageSubscription = (
       web3inboxClientData?.client.getSubscription(account, domain) ?? null
     );
 
+  const [watching, setWatching] = useState(false);
   const [error, setError] = useState<string | null>(null);
-
   const [isSubscribing, setIsSubscribing] = useState(false);
   const [isUnsubscribing, setIsUnsubscribing] = useState(false);
 
   useEffect(() => {
-    if (!web3inboxClientData) return;
+    if (!web3inboxClientData?.client || watching) return;
```

Comment:
Do we need to recall the `watchSubscription` even though we already started watching it?